src/backend/storeManagers/legendary/games.ts

```diff
@@ -888,24 +888,22 @@ export async function launch(
     ...getKnownFixesEnvVariables(appName, 'legendary')
   }
 
-  // We can get this env variable either from the game's settings or from known fixes
-  if (commandEnv['USE_FAKE_EPIC_EXE']) {
-    if (isWindows) {
-      commandEnv['LEGENDARY_WRAPPER_EXE'] = fakeEpicExePath
-    } else {
-      // on linux and mac, we copy the fake exe
-      const fakeExeWinPath = 'C:\\windows\\command\\EpicGamesLauncher.exe'
-      const fakeEpicExePathInPrefix = await getWinePath({
-        path: fakeExeWinPath,
-        gameSettings,
-        variant: 'unix'
-      })
+  // Use the wrapper EXE to launch games.
+  if (isWindows) {
```

Comment:
I wonder if we can make this a disable thing in the advanced settings of a game?

like enable it by default but keep the option somewhere to disable it just in case

I'm worried that this gets flagged sometimes by antivirus software and it will break games, and having the option to disable this would be useful

or maybe check if the file exists and it can be copied before adding the env variable? then if the antivirus deleted it it will be ignored by default too